administration/src/cards/extensions.tsx

```diff
@@ -60,16 +61,12 @@ export const birthday_extension: Extension<BirthdayState> = {
       <FormGroup label='Geburtsdatum'>
         <DateInput
           placeholder='Geburtsdatum'
-          value={daysSinceEpochToDate(state?.birthday || initialBirthdayDate)}
-          parseDate={value => new Date(value)}
-          onChange={value => {
-            if (!value) {
-              return setState(null)
-            }
-            setState({
-              birthday: dateToDaysSinceEpoch(value),
-            })
+          value={daysSinceEpochToDate(state?.birthday ?? initialBirthdayDate)}
+          parseDate={value => {
+            const millis = Date.parse(value)
+            return isNaN(millis) ? false : new Date(millis)
           }}
+          onChange={value => setState(value !== null ? { birthday: dateToDaysSinceEpoch(value) } : null)}
```

Comment:
These changes also fix, that the user was not able to set 1/1/1970 as a birthday.